Core/Object Arts/Dolphin/MVP/Presenters/List/ListBox.cls

```diff
@@ -17,6 +17,16 @@ Class Variables:
 !ListBox categoriesForClass!MVP-Views! !
 !ListBox methodsFor!
 
+anchorIndex
```

Comment:
The term "anchor" seems preferable to "selection mark". Perhaps we should just settle on this terminology and use the same selector names for ListView as well, that is if the anchor and selection mark behave in exactly the same way (I assume they do?). What do you think?